lib/typinggame_server/interactors/players/index.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'hanami/interactor'
+
+module Interactors
+  module Players
+    class Index
+      include Hanami::Interactor
+
+      expose :players
+
+      def initialize(dependencies = {})
+        @players_repository =
+          dependencies.fetch(:repository) { PlayersRepository.new }
```

Comment:
Having a default that references the dependency directly somewhat defeats the purpose of dependency injection in my opinion. Is this how Hanami wants us to do DI?

Reply:
@jgayfer No, this is adapted to my project from a tutorial, I'll look into how this works.

Reply:
@jgayfer Updated.